src/main/java/org/example/marketplacebackend/service/UserService.java

```diff
@@ -102,4 +102,12 @@ public boolean passwordChange(Account targetUser, String currentPasswordConfirma
   public Account saveUser(Account targetUser) {
     return accountRepo.save(targetUser);
   }
+
+  /**
+   * Deletes the given to user to the database.
+   * @param targetUser The user to be deleted.
+   */
+  public void deleteUser(Account targetUser) {
+    accountRepo.delete(targetUser);
+  }
```

Comment:
To delete a user this way requires that it isn't used as a foreign key in the database. When this is the case, this method should contain logic that removes these links in some way or alternatively we should add this removal logic to the database.